jp.nonbili.noutube.yml

```diff
@@ -0,0 +1,92 @@
+app-id: jp.nonbili.noutube
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '25.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node22
+command: run.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+modules:
+  - name: noutube
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/node22/bin:/run/build/noutube/bin
+      env:
+        VITE_SUPABASE_URL: https://pgukcvgypvjwtibzlvhr.supabase.co
+        VITE_SUPABASE_ANON_KEY: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InBndWtjdmd5cHZqd3RpYnpsdmhyIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MTI0NTIzODQsImV4cCI6MjAyODAyODM4NH0.zoxse4Kay_svHlQOiAINZm1lPIFPJMZAY8RKZUDSQrs
```

Comment:
how does embedding the token here work? is it safe on your end?

Reply:
`bun electron-vite build` will replace `import.meta.env.VITE_*` in the code with actual env values.
These are public info, any website connecting to supabase will contain supabase URL/ANON_KEY in every HTTP request